### PR TITLE
 add suggestions from PR #203 from branch timestap-without-timezone

### PR DESCRIPTION
**Pull request by raymondyin**
Created this new branch and PR for review on added suggestions from PR #203 from branch timestap-without-timezone, due to my pre-mature merge.

The added suggestions are in commit 0452746

### CODE REVIEW

**Comment by frankgh** [comment]
I think you want this commit in only in the branch 04527464b14859ba766e7f1dde18c875d12fdca5

File: server/pxf-service/src/main/java/org/greenplum/pxf/service/BridgeOutputBuilder.java [hunk 1]
```diff
@@ -257,9 +257,15 @@ void fillGPDBWritable(List<OneField> recFields) throws BadRecordException {
      * @return whether data type is string type
      */
     boolean isStringType(DataType type) {
-        return Arrays.asList(DataType.VARCHAR, DataType.BPCHAR, DataType.TEXT,
-                DataType.NUMERIC, DataType.TIMESTAMP, DataType.DATE).contains(
-                type);
+        return Arrays.asList(
```
Comment (lavjain) [inline server/pxf-service/src/main/java/org/greenplum/pxf/service/BridgeOutputBuilder.java]: Arrays.asList should be a static variable that is initialized only once.

Comment (raymondyin) [reply server/pxf-service/src/main/java/org/greenplum/pxf/service/BridgeOutputBuilder.java]: Thank you Lav. A new PR is created via another branch. Closing down this one as I didn't cherry-pick the target commit.

**Comment by raymondyin** [comment]
Closing due to improper branching. Please refer to PR #209 for the correct one.

**Pull request closed without merging**